dashboard/src/actions.ts

```diff
@@ -0,0 +1,145 @@
+import { Moment } from 'moment';
+
+import { RegionList, RegionStatsList } from './api/types';
+import { MapViewport } from './components/types';
+
+interface CheckExistingLoginAction {
+    type: 'CHECK_EXISTING_LOGIN';
+}
+export function checkExistingLogin(): CheckExistingLoginAction {
```

Comment:
Shorthand? It's possible to use lambdas too, but IMHO regular functions look nicer and I like to use them when possible. (The not-possible case is usually because the different this binding of lambdas makes it wiser to use lambdas for some this-using class methods.)

Reply:
> Shorthand?
I find the notication to be shorter and in most cases better then the `function ...` annotatation, but this should have been marked as bikeshed topic as well :)